contrib/epee/include/storages/parserse_base_utils.h

```diff
@@ -28,14 +28,8 @@
 
 #pragma once 
 
-#include <algorithm>
 #include <boost/utility/string_ref.hpp>
```

Comment:
`#include <string>` - one of the functions signatures references `std::string::const_iterator` which should require the entire definition for `std::string`.

Also looks like `#include <boost/utility/string_ref_fwd.hpp>` is now possible in this header.

Reply:
Both done.